client/rest-high-level/src/test/java/org/elasticsearch/client/SnapshotIT.java

```diff
@@ -55,6 +61,12 @@ private Response createTestSnapshot(String repository, String snapshot) throws I
         return highLevelClient().getLowLevelClient().performRequest(createSnapshot);
     }
 
+    private Response createTestSnapshot(String repository, String snapshot, String index) throws IOException {
+        Request createSnapshot = new Request("put", String.format(Locale.ROOT, "_snapshot/%s/%s", repository, snapshot));
+        createSnapshot.addParameter("wait_for_completion", "true");
+        createSnapshot.setEntity(new NStringEntity("{\"indices\":\""+index+"\"}", ContentType.APPLICATION_JSON));
+        return highLevelClient().getLowLevelClient().performRequest(createSnapshot);
+    }
```

Comment:
You should rebase on 90d62e6. It included a `createTestSnapshot` method here as well for his tests.